data_import/dilatometry_importer.py

```
@@ -0,0 +1,72 @@
+            'relative_change': df['relative_change'].values,
+            'differential_change': df['differential_change'].values
+        }
+
+        return result
+
+    except FileNotFoundError:
+        logger.error(f"File not found: {file_path}")
+        raise
+    except Exception as e:
+        logger.error(f"Error importing dilatometry data: {str(e)}")
+        raise ValueError(f"Unable to import dilatometry data. Error: {str(e)}")
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)